python/paddle/nn/functional/activation.py

```diff
@@ -634,24 +658,24 @@ def rrelu(x, lower=1.0 / 8.0, upper=1.0 / 3.0, training=True, name=None):
     Examples:
         .. code-block:: python
 
-            import paddle
-            import paddle.nn.functional as F
-
-            input_tensor = paddle.to_tensor([[[[-2.0,  3.0, -4.0,  5.0],
-                                            [ 3.0, -4.0,  5.0, -6.0],
-                                            [-7.0, -8.0,  8.0,  9.0]],
-                                            [[ 1.0, -2.0, -3.0,  4.0],
-                                            [-5.0,  6.0,  7.0, -8.0],
-                                            [ 6.0,  7.0,  8.0,  9.0]]]], dtype='float32')
-
-            out = F.rrelu(input_tensor, 0.1, 0.3)
-            print(out)
-            #[[[[-0.20000899  3.         -0.8810822   5.        ]
-            #   [ 3.         -0.55175185  5.         -1.0776101 ]
-            #   [-1.0680687  -1.9896201   8.          9.        ]]
-            #  [[ 1.         -0.5238267  -0.65515125  4.        ]
-            #   [-1.3766339   6.          7.         -2.3465784 ]
-            #   [ 6.          7.          8.          9.        ]]]]
+            >>> import paddle
+            >>> import paddle.nn.functional as F
+
+            >>> input_tensor = paddle.to_tensor([[[[-2.0,  3.0, -4.0,  5.0],
+            ...                                    [ 3.0, -4.0,  5.0, -6.0],
+            ...                                    [-7.0, -8.0,  8.0,  9.0]],
+            ...                                   [[ 1.0, -2.0, -3.0,  4.0],
+            ...                                    [-5.0,  6.0,  7.0, -8.0],
+            ...                                    [ 6.0,  7.0,  8.0,  9.0]]]], dtype='float32')
+            >>> out = F.rrelu(input_tensor, 0.1, 0.3)
```

Comment:
`rrelu`这个api也带有随机性，参考[rrelu API文档](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/rrelu_cn.html#rrelu)

- 加seed
- 下面的输出同步修改

```suggestion
            >>> paddle.seed(2023)
            >>> out = F.rrelu(input_tensor, 0.1, 0.3)
```